axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceServiceImpl.java

```diff
@@ -854,4 +857,19 @@ public Boolean checkPartnerBankDetailsList(Invoice invoice) {
 
     return true;
   }
+  
+  protected boolean checkEnablePDFGenerationOnVentilation(Invoice invoice) throws AxelorException 
+  {
+	  	//isPurchase() = isSupplier()
+		if(appAccountService.getAppInvoice().getIsEnablePDFGenerationForCustomersOnVentilation() && !InvoiceToolService.isPurchase(invoice)) 
+		{
```

Comment:
Indentation is not good. Please apply google code format.

Reply:
Done